tests/neo.UnitTests/protocol.json

```diff
@@ -1,5 +1,5 @@
 {
   "ProtocolConfiguration": {
-    "MillisecondsPerBlock": 2000
+    "MillisecondsPerBlock": 200
```

Comment:
Is there any risk of using a lower value? If this is for UT only, we could try to use the smallest value possible.

Reply:
I think 200 is enough. It was good for our needs. 

If someone have a use with lower value it should be decreased.